### ISSUE TITLE
SFM in Windows 11

### ISSUE THREAD

**Issue by Ansh4011** [aleksey-hoffman/sigma-file-manager#162]
SFM looks awesome with curved edges of Windows 11! Is there any chance to get the window maximizer option which allows us to select where do we want to position the windows to by hovering over toggle window size? 
![image](https://user-images.githubusercontent.com/84930340/136549660-94686177-7066-4304-acd1-68cf12828cbd.png)
![image](https://user-images.githubusercontent.com/84930340/136549470-c915951a-092c-45fd-a417-a568b317b008.png)

**Comment by aleksey-hoffman** [comment]
Yes, it is possible. Electron developers already [merged this](https://github.com/electron/electron/pull/29600) in Electron v13, v14, v15. 
SFM is currently using Electron v12, I will upgrade it to v15 for the next update. though I'm not sure if this "Windows controls" feature will be added in the next update
